packages/cli/internal/pkg/cli/logs_core.go

```diff
@@ -132,10 +136,15 @@ func (o *logsSharedOpts) displayEventFromChannel(channel <-chan cwl.StreamEvent)
 			for _, line := range event.Logs {
 				printLn(line)
 			}
+			firstEvent = false
+		} else if firstEvent {
+			firstEvent = false
```

Comment:
Can this code duplication be avoid?

Reply:
We could set the value once for every log event but that is less efficient since it would trigger every second. I can change it if required but omitted it consciously.